rosdep/base.yaml

```diff
@@ -482,6 +482,7 @@ collada-dom:
   ubuntu:
     artful: [libcollada-dom2.4-dp-dev]
     bionic: [libcollada-dom2.4-dp-dev]
+    focal: [libcollada-dom2.4-dp-dev]
```

Comment:
At this point, the name of this key for all supported Ubuntu distributions is `libcollada-dom2.4-dp-dev`.  For future compatibility, I think we can collapse this whole key into:

```
ubuntu: [libcollada-dom2.4-dp-dev]
```